.github/workflows/ci.yml

```diff
@@ -45,11 +43,14 @@ jobs:
       - name: Build
         run: make all
 
-      - name: Upload build artifacts
+      - name: Bundle Artifacts
+        run: tar -cvzf bluejay_firmware.tar.gz -C build/hex/ .
```

Comment:
It would be nice if we could wrap the resulting hex files in a directory with the run number:

```
run: |
  mkdir "bluejay-${GITHUB_RUN_NUMBER}"
  cp build/hex/*.hex "bluejay-${GITHUB_RUN_NUMBER}"
  tar -cvzf "bluejay-${GITHUB_RUN_NUMBER}.tar.gz" "bluejay-${GITHUB_RUN_NUMBER}"
```